core/src/test/java/org/apache/iceberg/TestMetadataUpdateParser.java

```diff
@@ -1371,7 +1371,6 @@ private void assertEqualsRemoveEncryptionKey(
   private String createManifestListWithManifestFiles(long snapshotId, Long parentSnapshotId)
       throws IOException {
     File manifestList = File.createTempFile("manifests", null, temp.toFile());
```

Comment:
we should also update this one to `File manifestList = temp.resolve("manifests").toFile();` so that we're doing the same thing across the codebase